tests/knative-build/knative_build_test.go

```diff
@@ -0,0 +1,104 @@
+package knative_build_acceptance
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/avast/retry-go"
+	build_api "github.com/knative/build/pkg/apis/build/v1alpha1"
+	build "github.com/knative/build/pkg/client/clientset/versioned/typed/build/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	cpuLimits = "50m"
+)
+
+func TestKnativeBuild_Acceptance(t *testing.T) {
+	kubeConfig := loadKubeConfigOrDie()
+	buildClient := build.NewForConfigOrDie(kubeConfig).Builds("knative-build")
+	labels := make(map[string]string)
+	labels["test-build-func"] = "test-build"
+	build, err := buildClient.Create(&build_api.Build{
+		ObjectMeta: meta.ObjectMeta{
+			Name:      "test-build",
+			Labels:    labels,
+			Namespace: "knative-build",
+		},
+		Spec: build_api.BuildSpec{
+			Steps: []corev1.Container{{
+				Name:    "test-build",
+				Image:   "alpine:3.8",
+				Command: []string{"echo", "-n", "hello build"},
+			}},
+		},
+	})
+
+	if err != nil {
+		t.Fatalf("Unable to create build: %v", err)
+	}
+
+	defer deleteBuild(buildClient, build)
+
+	err = retry.Do(func() error {
+		log.Printf("Checking build logs")
+		cmd := exec.Command("kubectl", "-n", "knative-build", "logs", "-l", "test-build-func=test-build", "-c", "build-step-test-build")
```

Comment:
maybe we should read the logs using client-go as well. then we could get rid off the kubectl binary in the container

Reply:
fixed.